components/base/Typography.tsx

```diff
@@ -136,6 +137,50 @@ export const listStyles = css`
   }
 `;
 
+// Table styles
+export const tableStyles = css`
+  border-spacing: 0;
+
+  thead {
+    th {
+      font-family: ${ROBOTO};
+      font-weight: 500;
+      font-size: 13px;
+      color: ${HEADLINE_TEXT};
+      line-height: 17px;
+      padding: 8px 16px;
+    }
+  }
+
+  tbody {
+    tr {
+      box-shadow: 0 0 0 1px ${GREY_2}; /* https://stackoverflow.com/a/48831092 */
+
+      &:nth-of-type(1) {
```

Comment:
I initialled had `:first-child` here, but Emotion was complaining about it potentially being an issue with SSR (🤷) so I changed it to this.